charts/opensearch/values.yaml

```diff
@@ -47,6 +52,7 @@ config:
     network.host: 0.0.0.0
 
     # Setting network.host to a non-loopback address enables the annoying bootstrap checks. "Single-node" mode disables them again.
+    # Requires ".singleNode" to be "true"
     # discovery.type: single-node
```

Comment:
Does this have an environment variable that could be used instead?

Reply:
Yes, the `discovery.type` environment variable seems to set this as well. I updated the PR.